portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -140,9 +140,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   }
 
-  private int checkUnallowedInput(String argURL, String argComment)
+  public int checkUnallowedInput(String argURL, String argComment)
```

Comment:
You can make it a package access method, because test is of the same package.
```suggestion
  int checkUnallowedInput(String argURL, String argComment)
```